client/src/components/Search.jsx

```diff
@@ -199,15 +203,21 @@ const Search = (props) => {
     setCoopSearchSettings({...coopSearchSettings, [name]: selections})
   }
 
+  const handleToggle = (bool) => setListView(bool)
   // same logic from Search.jsx
   const renderSearchResults = () => {
+    console.log("is auth: " + isAuthenticated);
     if (searchResults && searchResults.length) {
+        if(listView){
+          return (
 
-         console.log(searchResults);
-         
-         return (
-           <RenderCoopList link={"/directory-additions-updates/"} searchResults={searchResults}  columnOneText={"Matching Entities"} columnTwoText={"Edit"} />
-         )
+            <RenderCoopList link={"/directory-additions-updates/"} searchResults={searchResults}  columnOneText={"Matching Entities"} columnTwoText={"Edit"} />
+          )
+        }else{
+          return (
+            <Spreadsheet searchResults={searchResults}/>
```

Comment:
What I had been thinking was we always render the "RenderCoopList" component (I'll have to change the name to just "CoopList" as the "Render" prefix seems redundant) but right above it, we have a link for authenticated users tahat reads "Edit in Spreadsheet" and clicking on that would whisk you away to the /spreadsheet page with the search results populated.

Reply:
Oh shoot -- did Steve say he wanted that spreadsheet view visible on the search page after a search is done?  Maybe I missed that.  Do you have a screen shot of your local that you could email me of doing a search and how it would appear?  We can get confirmation from him via email.

Reply:
I might have misunderstood the requirements too, so no worries! 

- I just emailed you screenshots of the current state. 
- In the meantime, I've already started working on a version that routes you to the spreadsheet page in another branch.

Reply:
Resolving, Steve confirmed the way you have it is how we want to go.